speed/inrix/sf_inrix_xd_realtime.py

```diff
@@ -20,12 +39,16 @@
     spatial_coverage_thresholds = [99, 95, 90, 85, 80, 75, 70]
 
     zip_filepaths = glob_inrix_xd_zips_directory(
-        config["input"]["dir"], config["input"]["glob"]
+        config["input"]["dir"],
+        f'{config["input"]["year"]}/{config["input"]["month"]}/*{config["input"]["end_date"]}*.zip',
     )
 
     read_and_process_speed_to_los_and_reliability(
         zip_filepaths,
-        Path(config["output"]["dir"], config["output"]["filename_stem"]),
+        Path(
+            f'{config["output"]["base_dir"]}/{config["output"]["version"]}',
```

Comment:
Ah I see that filename_stem is probably "AutoSpeeds". In that case, should we just hardcode it? and have `filename_stem = f'{config["output"]["version"]}/{config["input"]["year"]}{config["input"]["month"]}_AutoSpeeds` ? (since it seems like we always use it together with the version dir. I'm not sure how this might work with the json line below of the `"\\"`. Does using a forward slash work?

Reply:
Will modify it.